plugin.video.tyt/main.py

```diff
@@ -19,247 +18,127 @@
-               "Aggressive Progressives" : {"url":ap_url, "thumb":ap_thumb, "type":"members"},
-               "The News with Dan Rather": {"url":danrather_url, "thumb":danrather_thumb, "type":"members"},
-               "Rebel Headquarters" : {"url":rebelhq_url, "thumb":rebelhq_thumb, "type":"members"},
-               "Post Game":         {"url":pg_url, "thumb":pg_thumb, "type":"members"},
-               "Old School":        {"url":oldschool_url, "thumb":oldschool_thumb, "type":"members"},
-               "TYT Classics":      {"url":tytclassics_url, "thumb":tytclassics_thumb, "type":"members"},
-               "Murder with Friends": {"url":murder_url, "thumb":murder_thumb, "type":"members"},
-               "Special Events":    {"url":special_events_url, "thumb":special_events_thumb, "type":"members"},
-               "Members LIVE show": {"url":members_live_url, "thumb":members_live_thumb, "type":"members_live"},
-               "Behind The Scenes": {"url":bts_url, "thumb":bts_thumb, "type":"members"}}
-              
-
-main_cat = {   "Members Only":      {"menu":"members", "thumb":hour1_thumb, "type":"members_menu"},
-               "TYT Sports":        {"url":tytsports_url, "thumb":tytsports_thumb, "type":"youtube_channel"},
-               "TYT Politics":      {"url":tytpolitics_url, "thumb":tytpolitics_thumb, "type":"youtube_channel"},
-               "What The Flick?!":  {"url":wtf_url, "thumb":wtf_thumb, "type":"youtube_channel"},
-               "Pop Trigger":       {"url":pop_url, "thumb":pop_thumb, "type":"youtube_channel"},
-               "TYT Interviews":    {"url":tytint_url, "thumb":tytint_thumb, "type":"youtube_channel"},
-               "Nerd Alert":        {"url":nerd_url, "thumb":nerd_thumb, "type":"youtube_channel"},
-               "Secular Talk":      {"url":sec_url, "thumb":sec_thumb, "type":"youtube_channel"},
-               "The Point with Ana Kasparian": {"url":point_url, "thumb":point_thumb, "type":"youtube_channel"},
-               "The Majority Report with Sam Seder": {"url":sam_url, "thumb":sam_thumb, "type":"youtube_channel"},
-               "TYT Live Stream":   {"url":tytlive_url, "thumb":tytlive_thumb, "type":"youtube_video"},
-               "ThinkTank":         {"url":thinktank_url, "thumb":thinktank_thumb, "type":"youtube_channel"}}
-menus = {"main":main_cat,"members":members_cat}
+MEDIA_URL = 'special://home/addons/{0}/resources/images/'.format(PLUGIN_ID)
 
 def show_changelog():
-  #with open(settings.getAddonInfo('changelog')) as f:
   with open(changelog) as f:
     text = f.read()
   dialog = xbmcgui.Dialog()
   label = '%s - %s' % (xbmc.getLocalizedString(24054), settings.getAddonInfo('name'))
   dialog.textviewer(label, text)
 
-def jw_website(page):
-  f = urllib2.urlopen(page)
-  return f.read()
-  
-def sendResponse(cookies, pagename): 
-  conn = httplib.HTTPSConnection("tytnetwork.com")
+def sendResponse(cookies, pagename):
+  conn = httplib.HTTPSConnection("tyt.com")
```

Comment:
minor suggestion: usually a good practice to have the server on a variable you can change without messing with the main code

Reply:
Left as is.  Unlikely that the website will change.